gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -71,21 +81,20 @@ def run(argv=None):
                                                           _COMMAND_LINE_OPTIONS)
   options = pipeline_options.PipelineOptions(pipeline_args)
   google_cloud_options = options.view_as(pipeline_options.GoogleCloudOptions)
-  # TODO(allieychen): Add support for local location.
-  if not google_cloud_options.temp_location or not google_cloud_options.project:
-    raise ValueError('temp_location and project must be set.')
 
+  temp_folder = google_cloud_options.temp_location or tempfile.mkdtemp()
   timestamp_str = datetime.now().strftime('%Y%m%d_%H%M%S')
   vcf_data_temp_folder = filesystems.FileSystems.join(
-      google_cloud_options.temp_location,
+      temp_folder,
       'bq_to_vcf_data_temp_files_{}'.format(timestamp_str))
+  filesystems.FileSystems.mkdirs(vcf_data_temp_folder)
```

Comment:
This line is needed when using direct runner. Otherwise, It won't be able to create the file if the directory is not exist.